recipe/meta.yaml

```diff
@@ -1,48 +1,62 @@
 {% set name = "starlette" %}
-{% set version = "0.17.0" %}
+{% set version = "0.27.0" %}
 
 package:
   name: {{ name|lower }}-recipe
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 31a889e7d7bf487f70d9d197ed7efadb47fa938c58626ed93e381480833c5b84
+  sha256: 6a6b0d042acb8d469a01eba54e9cda6cbd24ac602c4cd016723117d6a7e73b75
 
 build:
   number: 0
+  skip: true  # [py<37]
+
+requirements:
+  host:
+    - python
+    - pip
+    - hatchling
```

Comment:
I think this section (the top level build ands requirements) isn't needed.

Reply:
It's a conda-build bug. Without this section, the build will fail